src/__tests__/CompareResults/SubtestsRevisionRow.test.tsx

```diff
@@ -98,6 +101,9 @@ describe('SubtestsRevisionRow Component', () => {
       />,
     );
 
+    // wait until all elements render correctly
+    await screen.findByText('dhtml.html');
```

Comment:
It is even more important in this test.

Indeed in the previous test, we were using `await` until the text for either `firefox` or `chrome` was showing up. So at the end of the test, the component was rendered.

But in _this_ test, we're using `queryByText` which doesn't wait. So:
1. Indeed the page wasn't rendered at all, so of course the text wasn't there,
2. but also the page was still being rendered _after_ the test. This is what triggered the warning about `act`: something was rendered when the test wasn't waiting anything.

Reply:
Okay this makes sense